### ISSUE TITLE
Lint/SymbolConversion false positive for Hash keys that contains ":"

### ISSUE THREAD

**Issue by eduardoprentcomau** [rubocop-hq/rubocop#9484]
## Expected behavior

Describe here how you expected RuboCop to behave in this particular situation.

## Actual behavior

Rubocop trying to convert a hash key containing a colon instead of ignoring it.
If a key contains two points, rubocop tries to convert it as if it were a symbol.

## Steps to reproduce the problem

This is extremely important! Providing us with a reliable way to reproduce
a problem will expedite its solution.

1. Create a sample file with the content:
```ruby
# frozen_string_literal: true

{
  'some:key': 'value',
}
```
2. Execute the rubocop validation to the file
3. Should return the error: `W: [Correctable] Lint/SymbolConversion: Unnecessary symbol conversion; use "somekey": instead.`

## RuboCop version

```
$ [bundle exec] rubocop -V
1.9.0 (using Parser 3.0.0.0, rubocop-ast 1.4.1, running on ruby 2.7.2 x86_64-linux-musl)
  - rubocop-rails 2.9.1
  - rubocop-rspec 2.1.0
```

**Comment by eduardoprentcomau** [comment]
Sorry guys, duplicated from: https://github.com/rubocop-hq/rubocop/issues/9455

**Comment by koic** [comment]
Bug fix release is planned for RuboCop 1.9. Please wait a moment. Thank you!
https://github.com/rubocop-hq/rubocop/pull/9476#issuecomment-770261115
